crates/subspace-node/src/chain_spec.rs

```diff
@@ -166,17 +163,14 @@ pub fn subspace_testnet_config() -> Result<SubspaceChainSpec, String> {
     ))
 }
 
-pub fn subspace_development_config() -> Result<SubspaceChainSpec, String> {
-    use subspace_runtime::SSC;
-
-    let wasm_binary = subspace_runtime::WASM_BINARY
-        .ok_or_else(|| "Development wasm not available".to_string())?;
+pub fn dev_config() -> Result<SubspaceChainSpec, String> {
+    let wasm_binary = WASM_BINARY.ok_or_else(|| "Development wasm not available".to_string())?;
 
     Ok(SubspaceChainSpec::from_genesis(
         // Name
-        "Development",
+        "Subspace development",
         // ID
-        "dev",
+        "subspace-dev",
```

Comment:
Make it consistent if you decide to change it.

```suggestion
        "subspace_dev",
```

Reply:
prefer `subspace-dev` 🤷🏼

Reply:
Then `subspace-local `, I don't have a strong opinion, just be consistent :P

Reply:
Changed to underscore as this is what Substrate and our testnet are using